src/components/Search/Search.js

```diff
@@ -204,6 +208,7 @@ Search.defaultProps = {
   placeholder: '',
   filters: [],
   onPin: noop,
+  onFilterSelect: noop,
```

Comment:
Wonder if we could make `addSearchTerm` a default callback here. Or else I'd make it a required prop, wouldn't make sense to let the user go without specifying this callback, otherwise the filters would just not work ;)

Reply:
Interesting thought! Replacing default behaviour sounds confusing, I wonder if you can declare dependent proptypes...